crates/extensions/az_mapper_ext/src/converter.rs

```diff
@@ -48,11 +50,26 @@ impl AzureConverter {
     }
 
     fn try_convert(&mut self, input: &MqttMessage) -> Result<Vec<MqttMessage>, ConversionError> {
+        let default_timestamp = self.add_timestamp.then(|| self.clock.now());
         if is_bridge_health(&input.topic.name) {
             Ok(vec![])
+        } else if input.topic.name.starts_with("tedge/health") {
```

Comment:
This check is too wild. The mapper subscribes to `tedge/health/+` and `tedge/health/+/+`. Therefore, for example, I think this c8y message will be translated by the azure mapper.

```
tedge mqtt pub tedge/health/tedge-mapper-c8y '{"pid":19753,"status":"up","time":1694586060}'
```

Reply:
@rina23q Though shouldn't the az mapper forward all service status messages that are published in the json format? It should not care which service is actually behind it, as you can use Azure to monitor the c8y service.

Reply:
@reubenmiller Ah okay, but then simple forwarding the payload `{"pid":19753,"status":"up","time":1694586060}` to `az/messages/events/` is not enough. How can user know which service is behind the message?

Honestly speaking, I'm still confused what is preferred conversion output for azure.

Reply:
It may be for another PR, but I feel maybe we can use the `{property-bag}` messages to send extra information. Find more info [here](https://learn.microsoft.com/en-us/azure/iot/iot-mqtt-connect-to-iot-hub#sending-device-to-cloud-messages).

Reply:
Yes we can do the adjustments to how to communicate which service it is in a follow service. Though someone needs to point some thought into how to do this following the best practices of Azure IoT Hub.